### ISSUE TITLE
Units of the respiration belt

### ISSUE THREAD

**Issue by celprov** [TheAxonLab/hcph-sops#409]
I've just had a chat with Stefano, and he says that BioPack measures and outputs signals in mV.
Did we do some transformation to the respiration belt signal to convert it to cm H20 unit (cf physio.json)?

**Comment by oesteban** [comment]
@smoia is wrong on this one :)

With BIOPAC, if you properly select the transducer, the calibration outputs the pressure in cm H20 units (meaning 1 volt is 1 cm or similar, I would need to check how this is calibrated exactly). This is true for at least our settings - maybe older versions were not calibrated this way.

**Comment by smoia** [comment]
It's possible that in the newer system they make the conversion automatically between V and cm H2O.

What I was referring to is that the measure per se is not cm H2O (that's a conversion the system does), so I'd normally suggest to report the results in Volts with the specific (linear) transformation formula that can be used, if wanted, to make such conversion.

This is for two reasons:
1 - cm H20 is not a measure typically used in our field, that, having a background strongly rooted in US/UK/CA, unfortunately uses imperial units. Most of the time, you'd find yourself converting once again, so for that much, Volts are fine.
2 - Besides RVT and PetCO2/PetO2, most of our models for respiratory assessment end up being unitless.

(On top of that, there's a lot of variability from transducer to transducer and from person to person, but if you are 100% that your measures are precise, then no problem there ;) )

**Comment by oesteban** [comment]
> What I was referring to is that the measure per se is not cm H2O (that's a conversion the system does), so I'd normally suggest to report the results in Volts with the specific (linear) transformation formula that can be used, if wanted, to make such conversion.

I know ;) - the system only outputs electric potentials, so it can only be volts (physically).

However, we're talking here about BIDS. When converting into BIDS, the best option is not to manipulate the data in any way. That means, if AcqKnowledge says this is H2O cm and BIDS allows any unit (which is the case), then you just store that unit. An example where conversion of units must happen is ms, as time is reported in s within BIDS.

> This is for two reasons: 1 - cm H20 is not a measure typically used in our field, that, having a background strongly rooted in US/UK/CA, unfortunately uses imperial units. Most of the time, you'd find yourself converting once again, so for that much, Volts are fine. 2 - Besides RVT and PetCO2/PetO2, most of our models for respiratory assessment end up being unitless.
> 
> (On top of that, there's a lot of variability from transducer to transducer and from person to person, but if you are 100% that your measures are precise, then no problem there ;) )

I totally agree with this - but this is relevant in reporting. BIDS just wants the data to be shared in a pre-specified way that makes the user as close as possible to having acquired the data themselves.

**Comment by oesteban** [comment]
> I know ;)

NOTE: I admittedly wanted to poke a little at you, please don't take it the wrong way bc intention is good. I'm the child and you are the master on these topics -- without your feedback we would not be even talking about this (we would not have managed to pull this out, in total honesty)

**Comment by smoia** [comment]
Hey, we're learning together. In fact with the new year we'll probably discuss BIDS extension proposals within physiopy to further discuss these topics and propose a more decent standardisation of the data.

> However, we're talking here about BIDS. When converting into BIDS, the best option is not to manipulate the data in any way. That means, if AcqKnowledge says this is H2O cm and BIDS allows any unit (which is the case), then you just store that unit. An example where conversion of units must happen is ms, as time is reported in s within BIDS.

I think we agree on the principle of no manipulation here, where we see it differently is whether the (linear) transformation AcqKnowledge performs in terms of V → cm H2O is a manipulation or not.
Is there any clear algorithm reported anywhere from AcqKnowledge? If not, I'd much rather be cautious, and report Volts. But that's me, it does not have to be you ;)

**Comment by oesteban** [comment]
> Is there any clear algorithm reported anywhere from AcqKnowledge? If not, I'd much rather be cautious, and report Volts. But that's me, it does not have to be you ;)

Yes, you can workaround the "auto" calibration by saying that the transducer is "custom". If you do that, then you decide how you do the mapping. However we \*did not\*, so our data are already in whatever automagical manipulation AcqKnowledge did (unfortunately).

**Comment by oesteban** [comment]
In other words, it is too late for us to do the right thing, which is what you suggest.
